starboard/raspi/shared/launcher.py

```diff
@@ -126,31 +132,43 @@ def __init__(self, platform, target_name, config, device_id, **kwargs):
 
     self.last_run_pexpect_cmd = ''
 
+  def _GetAndCheckTestFile(self, target_name):
+    # TODO(b/218889313): This should reference the bin/ subdir when that's
+    # used.
+    test_dir = os.path.join(self.out_directory, 'install', target_name)
+    test_file = target_name
+    test_path = os.path.join(test_dir, test_file)
+
+    if not os.path.isfile(test_path):
+      raise TargetPathError(f'TargetPath ({test_path}) must be a file.')
+    return test_file
+
+  def _GetAndCheckTestFileWithFallback(self):
```

Comment:
can this be abstracted out for all launchers ?

Okay to do it in a separate cl